x/simulation/params.go

```diff
@@ -142,3 +145,23 @@ func (w WeightedProposalContent) DefaultWeight() int {
 func (w WeightedProposalContent) ContentSimulatorFn() simulation.ContentSimulatorFn {
 	return w.contentSimulatorFn
 }
+
+//-----------------------------------------------------------------------------
+// Param change proposals
+
+// RandomParams returns random simulation consensus parameters
+func RandomConsensusParams(r *rand.Rand) *abci.ConsensusParams {
+	return &abci.ConsensusParams{
+		Block: &abci.BlockParams{
+			MaxBytes: int64(simulation.RandIntBetween(r, 20000000, 30000000)),
+			MaxGas:   -1,
+		},
+		Evidence: &abci.EvidenceParams{
+			MaxAgeNumBlocks: int64(simulation.RandIntBetween(r, 100000, 500000)),
+			MaxAgeDuration:  time.Duration(simulation.RandIntBetween(r, 10000, 50000)),
+		},
+		Validator: &abci.ValidatorParams{
+			PubKeyTypes: []string{"secp256k1"},
```

Comment:
I wouldn't let this be random. Take a look at the simulation setup. We allow SECP keys and ED25519 keys.